modules/tax/src/TaxTypeRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\commerce_tax;
+
+use CommerceGuys\Tax\Repository\TaxTypeRepositoryInterface;
+
+/**
+ * Manages tax types based on JSON definitions.
+ */
+class TaxTypeRepository implements TaxTypeRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function get($id)
+    {
+        return entity_load('commerce_tax_type', $id);
```

Comment:
entity_load* functions are deprecated and to be removed from core soon. There are two ways to load an entity in D8:
1) Use the static method on the entity class. For example Node::load(), Product::load(), etc. This is mostly used in cases where dependency injection is impossible (procedural code, form api callbacks)
2) Inject the entity storage class. Plenty of examples of that in Commerce

In this case we need to do #2.